src/cider/nrepl/middleware/debug.clj

```diff
@@ -438,12 +438,28 @@
             (eval form)
             (alter-meta! v assoc ::instrumented instrumented)))))))
 
+(def step-in-blacklist
+  "A set of names or regexes specifying namespaces that should not be stepped in
+  to. By default, this contains only \"clojure.core\"."
+  (atom #{"clojure.core"}))
+
+(defn blacklisted-for-step-in?
```

Comment:
I don't think this is specific to `step-in` and I prefer positive predicates when possible. So rename this to `instrumentable-namespace?` and negate the return value.